b827ebfffefb5174.JSON

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "eui-b827ebfffefb5174",
```

Comment:
The naming is confusing, but this field should contain the EUI actually. So, instead of the lower case `eui-b827ebfffefb5174`, it should be the upper case without the prefix: `B827EBFFFEFB5174`.